src/components/ui/Dialog.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react";
+import * as DialogPrimitive from "@radix-ui/react-dialog";
+import { X } from "lucide-react";
+
+import { cn } from "@/lib/utils";
+
+const Dialog = DialogPrimitive.Root;
+
+const DialogTrigger = DialogPrimitive.Trigger;
+
+const DialogPortal = ({
+  className,
+  children,
+  ...props
+}: DialogPrimitive.DialogPortalProps) => (
+  <DialogPrimitive.Portal className={cn(className)} {...props}>
+    <div className="fixed inset-0 z-50 flex items-start justify-center sm:items-center">
+      {children}
+    </div>
+  </DialogPrimitive.Portal>
+);
+DialogPortal.displayName = DialogPrimitive.Portal.displayName;
+
+const DialogOverlay = React.forwardRef<
+  React.ElementRef<typeof DialogPrimitive.Overlay>,
+  React.ComponentPropsWithoutRef<typeof DialogPrimitive.Overlay>
+>(({ className, ...props }, ref) => (
+  <DialogPrimitive.Overlay
+    ref={ref}
+    className={cn(
+      "bg-background/80 fixed inset-0 z-50 backdrop-blur-sm transition-all duration-100 data-[state=closed]:animate-out data-[state=closed]:fade-out data-[state=open]:fade-in",
```

Comment:
I think the background for the dialog is a bit too opaque. Could we lower the opacity for it ? (I'm guessing it's the bg-background/80 class)

Reply:
Is 60 ok?

![image](https://user-images.githubusercontent.com/53144702/232713343-dc3d2546-4dfe-4ae9-b58a-7d6e0fd18fd9.png)

Reply:
Try even lower 😅

Reply:
Can also go for 30 :)

![image](https://user-images.githubusercontent.com/53144702/232713956-99807eca-2061-41f5-be8a-9c298cd31f19.png)

Reply:
Let's go with 40 and merge this 👌

Reply:
Pushed :)

Prettier also reordered classes for whatever reason :shrug: